catalog.html

```diff
@@ -0,0 +1,328 @@
+        Санкт-Петербург, ул. Б. Конюшенная, д. 19/8 
+      </p>
+      <div>
+        <a href="#">
+          <span>Личный кабинет</span>
+          <p>Равшан Джамшутович</p>
+        </a>
+        <a href="#"><span>Выйти</span></a>
+        <p>
+          <a href="#">Мои заказы</a>
+          <a href="#">Личный кабинет</a>
+        </p>
+      </div>
+      <nav>
+        <ul>
+          <li><a href="index.html">Главная</a></li>
+          <li><a href="#">Компания</a></li>
+          <li><a href="#">Каталог</a></li>
+          <li><a href="#">Новости</a></li>
+          <li><a href="#">Спецпредложения</a></li>
+          <li><a href="#">Доставка</a></li>
+          <li><a href="#">Контакты</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <ul>
+        <li>
+          <a href="index.html"><span>Главная страница</span></a>
+        </li>
+        <li>
+          <a href="#">Каталог</a>
+        </li>
+        <li>
+          <a href="#">Инструмент</a>
+        </li>
+        <li>Перфораторы</li>
+      </ul>
+      <h1>ПЕРФОРАТОРЫ</h1>
+      <section>
+        <h2>ФИЛЬТР:</h2>
+        <form action="https://echo.htmlacademy.ru/" method="GET">
+          <fieldset>
+            <legend>ЦЕНА:</legend>
+            <label for=prise-range-filter>Ценовой диапозон</label>
+            <input type="range" id="prise-range-filter" min="0"  value="30000">
+            <p>
+              <label for="min-prise-filter">Минимальная цена</label>   
+              <input type="number" name="min-prise" id="min-prise-filter" min="0" value="0">
+              -
+              <label for="max-prise-filter">Максимальная цена</label>
+              <input type="number" id="max-prise-filter" name="max-prise" min="0" value="30000">
+          </fieldset>
+          <fieldset>
+            <legend>ПРОИЗВОДИТЕЛИ:</legend>
+            <ul>
+              <li>
+                <input id="bosh-filter" name="bosh" type="checkbox" checked>
+                <label for="bosh-filter">BOSCH</label>
+              </li>
+              <li>
+                <input id="interskol-filter" name="interskol" type="checkbox">
+                <label for="interskol-filter">ИНТЕРСКОЛ</label>
+              </li>
+              <li>
+                <input id="makita-filter" name="makita" type="checkbox" checked>
+                <label for="makita-filter">MAKITA</label>
+              </li>
+              <li>
+                <input id="dewalt-filter" name="dewalt" type="checkbox">
+                <label for="dewalt-filter">Dewalt</label>
+              </li>
+              <li>
+                <input id="hitachi-filter" name="hitachi" type="checkbox">
+                <label for="hitachi-filter">Hitachi</label>
+              </li>
+            </ul>
+          </fieldset>
+          <fieldset>
+            <legend>ПИТАНИЕ:</legend>
+            <ul>
+              <li>
+                <input type="radio" name="power_supply" id="power_supply_battery" checked>
```

Comment:
```suggestion
                <input type="radio" name="power-supply" id="power-supply-battery" checked>
```

до этого ты использовала везде дефисы, поэтому неконсистентно теперь использовать подчеркивание

Reply:
поправь все вхождения